src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+html {
+  font-family: Roboto, serif;
+  font-weight: 400;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+@import './variables';
```

Comment:
Ensure that the `variables.scss` file exists and contains definitions for all the SCSS variables used in your project, such as `$border-radius`, `$primary-color`, `$secondary-color`, `$font-size-small`, `$font-size-large`, and `$button-bg`.